src/provider.go

```diff
@@ -86,6 +86,22 @@ func (p Provider) Get(s string) string {
 	case "vat_number":
 		// https://github.com/meanbee/magedbm2/blob/fc8bbf9a97db2c27d0cd8a1153dda8c95b6f5996/src/Anonymizer/Formatter/Company/VatNumber.php#L21
 		return "GB" + faker.Number().Between(100000000, 999999999)
+	case "blog_post_title":
+		return faker.Lorem().Sentence(6)
+	case "blog_post_slug":
+		return faker.Internet().Slug()
+	case "blog_post_content":
+		return faker.Lorem().Paragraph(3)
+	case "productcategoryname":
+		return faker.Commerce().Department()
+	case "productdescription":
+		return faker.Lorem().Paragraph(1)
+	case "productname":
+		return faker.Commerce().ProductName()
+	case "productsku":
+		return faker.Code().Abn()
+	case "shipment_tracking_number":
+		return faker.RandomString(18)
```

Comment:
Hey...Sorry for taking so long to respond. I'm sure you may have moved past this already. While I understand the shipment tracking number requirement, I'm not sure the use case of the other types. As there's nothing private or sensitive about a blog post name I'm not sure why this tool would ever need to anonymize something like that. This tool is intended for scrubbing data that shouldn't come out of a production environment such as PII

Reply:
We're using the tool for a slightly expanded purpose: taking a real client database and fully anonymizing it of both customer- and client-identifying information for use in a demo for prospective clients. That means also anonymizing the blog articles. We start with a real database because it provides a "shape" to the data that is more realistic than we could randomly generate (plus it's simpler).

Reply:
Gotcha. If you're interested in upgrading give v0.4.0 (which I just tagged) a try. I've implemented #10 in it so you should be able call Faker directly in your config to get the values you need.

You will not be able to call `faker.RandomString(18)` as the implementation is only compatible with struct methods, not package methods, but I've gone ahead and included "shipment_tracking_number" as I can see the value of providing that. All the other values you should be able to retrieve by specifying direct Faker function calls in your config.

I was hesitant to merge this as I don't want the config to have different values for `faker.Lorem.Paragraph(3)` and `faker.Lorem.Paragraph(1)` which could escalate into tons of variations for various functions with various length requirements.